defaults/main.yml

```diff
@@ -70,13 +70,10 @@ nextcloud_preview_folder_name: "preview-generator"
 nextcloud_preview_docker_folder: "/{{ nextcloud_preview_folder_name }}"
 nextcloud_preview_host_folder: "{{ nextcloud_customized_container_src_path }}/{{ nextcloud_preview_folder_name }}"
 nextcloud_preview_first_run_finished_filename: "finished-first-run.keepit"
-nextcloud_preview_squareSizes: "\"64 256 1024 2048\""
-nextcloud_preview_widthSizes: "\"64 256 1024 2048\""
-nextcloud_preview_heightSizes: "\"64 256 1024 2048\""
-nextcloud_preview_preview_max_x: 2048
-nextcloud_preview_preview_max_y: 2048
-nextcloud_preview_system_jpeg_quality: 60
-nextcloud_preview_app_jpeg_quality: "60"
+## Unsetting the preview_max variables will set them null in the config.yml (which is the dafult according to the docs).
```

Comment:
Typo

Reply:
This line of comment is dropped, so the typo is fixed